static/js/app.ts

```diff
@@ -2232,3 +2294,48 @@ const clearTimeouts = () => {
   timers.forEach(clearTimeout);
   timers = [];
 };
+
+export function downloadSlides(level: number) {
+  var iframe : any = document.getElementById(`level-${level}-slides`)!;
+  iframe.setAttribute('src',`/slides/${level}`);
+  $(`#level-${level}-slides`).on('load', function (){
+    var innerDoc = iframe.contentDocument || iframe.contentWindow.document;
+    var slides = innerDoc.getElementsByTagName('section');
+    var slidesHTML = ''
+    for (let i = 0; i < slides.length; i++) {
+      var innerIframe = slides[i].getElementsByTagName('iframe');        
+      for (let j = 0; j < innerIframe.length; j++) {
+        var a = document.createElement('a');          
+        a.href = 'https://www.hedycode.com' + innerIframe[j].getAttribute('src');
```

Comment:
Can we make this into hedy.org instead? A nicer url :)

Reply:
Ok!!